### PR TITLE
chore: contract method description

### PR DESCRIPTION
**Pull request by hanszhoou**
## Description

[MASK-632](https://mask.atlassian.net/browse/MASK-632)

From:

image

image

To:
image

image

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
  - [x] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Comment by hanszhoou** [comment]
@DimensionDev/qa

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
1. If tx is confirmed, the text doesn't work;
Screen Shot 2022-01-10 at 5 16 32 PM

Should be follow the Figma:
Screen Shot 2022-01-10 at 5 21 02 PM

2. For the NFT lucky drop, is also needed to change;

**Review by guanbinrui — DISMISSED** [review]

**Comment by wenluomask** [comment]
Typo: Create a NFT lucky drop
Screen Shot 2022-01-11 at 3 34 57 PM

**Comment by hanszhoou** [comment]
@wenluomask an NFT is right actually.

**Comment by wenluomask** [comment]
Yes you are right.
Checked.

File: packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/contractMethodDescription/getRedpacketDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    const { value: nativeToken } = useNativeTokenDetailed()
+    const { value: tokenDetailed } = useERC20TokenDetailed(tokenAddress)
+    const token = isNativeTokenAddress(tokenAddress) ? nativeToken : tokenDetailed
+
+    return (
+        <span>
+            {token && tokenAddress
+                ? t('plugin_red_packet_create_with_token', {
+                      symbol: `${formatBalance(tokenAmount, token.decimals)} ${token.symbol}`,
+                  })
```
Comment (septs) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/contractMethodDescription/getRedpacketDescription.tsx]:
```suggestion
                ? t('plugin_red_packet_create_with_token', {
                      balance: formatBalance(tokenAmount, token.decimals),
                      symbol: token.symbol,
                  })
```

File: packages/mask/shared-ui/locales/en-US.json [hunk 2]
```diff
@@ -358,13 +358,15 @@
     "plugin_red_packet_hint": "You can withdraw the remaining balance 24 hours after the Lucky Drop is sent.",
     "plugin_red_packet_token": "Token",
     "plugin_red_packet_message_label": "Title",
-    "plugin_red_packet_create_with_token": "Creating Lucky Drop with {{symbol}}",
+    "plugin_red_packet_create": "Create a Lucky Drop",
+    "plugin_red_packet_create_with_token": "Create a Lucky Drop with {{symbol}}",
```
Comment (septs) [inline packages/mask/shared-ui/locales/en-US.json]:
```suggestion
    "plugin_red_packet_create_with_token": "Create a Lucky Drop with {{balance}} {{symbol}}",
```

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**